venues/aclweb.org/Workshops/add-readers-to-original.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+from calendar import c
+from re import sub
+import openreview
+from tqdm import tqdm
+import csv
+
+
+"""
+OPTIONAL SCRIPT ARGUMENTS
+
+    baseurl -  the URL of the OpenReview server to connect to (live site: https://openreview.net)
+    username - the email address of the logging in user
+    password - the user's password
+    confid - the user's conference 
+
+"""
+parser = argparse.ArgumentParser()
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+parser.add_argument('--confid')
+args = parser.parse_args()
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+confid = args.confid
+#committee = [f'{confid}/Program_Chairs']
+#if openreview.tools.get_group(client, f'{confid}/Area_Chairs'):
+ #   committee.append(f'{confid}/Area_Chairs')
+# Retrieve all commitment submissions 
+commitment_notes = list(openreview.tools.iterget_notes(client, invitation = f'{confid}/-/Commitment_Submission'))
+for commitment_note in commitment_notes: 
+    paper_group = openreview.Group(
+        id = f'aclweb.org/ACL/2022/Conference/Paper{commitment_note.number}',
```

Comment:
I think so, I also seeing here that blind submissions are created, we don't need it for the workshops, right? only the PCs and authors can see the submisions

Reply:
yes, you are right!

Reply:
Don't use the word "Paper" to build the commitment group otherwise it will overlap with the regular submission groups. 

I would suggest to call it: `{venue_id}/Commitment{commitment_note.number}/Authors`